k8s/src/test/scala/io/buoyant/k8s/EndpointsNamerTest.scala

```diff
@@ -86,14 +111,13 @@ class EndpointsNamerTest extends FunSuite with Awaits {
         fail(s"unexpected request: $req")
     }
     val api = v1.Api(service)
-    val namer = new EndpointsNamer(Path.read("/test"), api.withNamespace(_))
+    val namer = new EndpointsNamer(Path.read("/test"), api.withNamespace)
 
     @volatile var state: Activity.State[NameTree[Name]] = Activity.Pending
-    val obs = namer.lookup(Path.read("/srv/thrift/sessions")).states respond { s =>
+    val _ = namer.lookup(Path.read("/srv/thrift/sessions")).states respond { s =>
```

Comment:
Good question.  Discarding values is a red-flag and usually not intentional.  I want to eventually turn on the compile option -Ywarn-value-discard which will cause a compiler error for discarded values.  In this case, though, we really do want to discard the return value from `respond`.  We can assign it to an unnamed variable to explicitly signal that discarding is intended here.

I'll make the same change on line 49.